ood.yaml

```diff
@@ -1,4 +1,7 @@
 ---
 - hosts: ood
   roles:
+    - prep_ood
     - ood
+    - warewulf_sync
+    - ood_firewall_and_services
```

Comment:
Similar to comment on prep_ood role name change motivation.  But in this case we might have two distinct operational roles:  ood_firewall_and_services (httpd, selinux, and firewall) and ohpc_firewall_and_services (munge and slurmd).